add-on/chrome/skin/shared/loop.css

```diff
@@ -310,6 +310,13 @@
   /****************************************************************/
   /* REMOTE CURSOR (keep synced with remote-cursor in common.css) */
   /****************************************************************/
+
+  #loop-remote-cursor-container {
+    position: absolute;
+    /* hide overflow so cursor does not push the DOM to the right/bottom*/
```

Comment:
I'd suggest making this more explicit to provide more context for readers new to the code.  Maybe something like "Hide overflow so that when the tail of the pointer gets to the edge of shared area, it doesn't widen the viewport by adding a scrollbar."